app/Services/Phoenix/Phoenix.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Rogue\Services\Phoenix;
+
+use DoSomething\Northstar\Common\RestApiClient;
+use DoSomething\Northstar\Resources\NorthstarClient;
+use DoSomething\Northstar\Resources\NorthstarUser;
+use DoSomething\Northstar\Resources\NorthstarUserCollection;
+use DoSomething\Northstar\Resources\NorthstarClientCollection;
+use GuzzleHttp\Cookie\CookieJar;
+use Cache;
+
+class Phoenix extends RestApiClient
+{
+	/**
+	* Create a new Phoenix API client.
+	* @param array $config
+	*/
+	public function __construct()
+	{
+        $this->base_uri = config('services.phoenix.uri') . '/api/' . config('services.phoenix.version') . '/';
+
+        $headers = [
+           'Content-type' => 'application/json',
+           'Accept' => 'application/json',
+        ];
+
+        parent::__construct($this->base_uri, $headers);
```

Comment:
I don't think that `RestApiClient` takes [more than one parameter](https://github.com/DoSomething/northstar-php/blob/821b6ebe02939b241c150ca672e306a37c54f3ed/src/Common/RestApiClient.php#L41) so I believe you can remove this second argument & the `$headers` array above.